client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get, invoke } from 'lodash';
+
+/**
+ * Internal Dependencies
```

Comment:
Uber nit. But also, consistency FTW 😛 

```suggestion
 * Internal dependencies
```